network-api/networkapi/utility/templatetags/environment_prefix.py

```diff
@@ -0,0 +1,15 @@
+from django import template
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag(takes_context=True)
+def environment_prefix(context):
```

Comment:
to make it easier to debug, let's make sure that the file itself does not have the same name as the tag function =)

Let's rename this to `mofo_common.py` and then it'll also be a good place for future utility tags that are site-wide applicable.

Reply:
Updated!